src/main/java/org/ilgcc/app/pdf/MultiProviderPDFService.java

```diff
@@ -192,7 +192,7 @@ Map<String, byte[]> generateAdditionalProviderPDF(Submission familySubmission) t
                             new SingleField("receivedTimestamp", formattedSubmittedAtDate, null));
                 });
 
-                additionalPDFs.put(getCCMSFileNameForAdditionalProviderPDF(familySubmission.getId(), i, providers.size()),
+                additionalPDFs.put(getCCMSFileNameForAdditionalProviderPDF(familySubmission.getId(), i, providerSchedulesByUuid.size()- 1),
```

Comment:
Very small nit just to add a space, otherwise this looks good!

```suggestion
                additionalPDFs.put(getCCMSFileNameForAdditionalProviderPDF(familySubmission.getId(), i, providerSchedulesByUuid.size() - 1),
```